src/components/structures/ContextMenu.tsx

```diff
@@ -614,7 +614,8 @@ export const useContextMenu = <T extends any = HTMLElement>(inputRef?: RefObject
         setIsOpen(false);
     };
 
-    return [button.current ? isOpen : false, button, open, close, setIsOpen];
```

Comment:
Would doing `button.current !==null ? isOpen...` achieve the same aim here? If so personally I'd find that more readable. I admit I haven't done much typing of refs but checking that it "isn't a function" seems counterintuitive to me. 

I'm not sure if this is to do with null checks, or not using coercion in checks, or a bit of both so I may have got the wrong end of the stick.